timetable/settings.py

```diff
@@ -72,16 +73,12 @@
 # https://docs.djangoproject.com/en/2.2/ref/settings/#databases
 
 DATABASES = {
-    'default': {
-        'ENGINE': 'django.db.backends.postgresql',
-        'NAME': os.environ.get('db_name', 'timetable'),
-        'USER': os.environ.get('db_user', 'postgres'),
-        'PASSWORD': os.environ.get('db_password'),
-        'HOST': '127.0.0.1',
-        'PORT': '5432'
-    }
+    'default': {}
 }
 
+DATABASES['default'] = dj_database_url.config(conn_max_age=500)
```

Comment:
Astea le poti pune in acelasi config ca is 2 parametrii. Si le poti muta mai sus in dictionarul ala n-are rost sa mai accesezi odata de aici. After that e good to go daca decidem ca cea default e cea remote :D